pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -294,7 +294,7 @@ func (c *Cluster) Start() error {
 	}
 
 	if c.spec.Storage.Store.UpdateStore == OSDStoreUpdateConfirmation {
-		delOpts := &k8sutil.DeleteOptions{MustDelete: true, WaitOptions: k8sutil.WaitOptions{Wait: true}}
+		delOpts := &k8sutil.DeleteOptions{WaitOptions: k8sutil.WaitOptions{Wait: true}}
```

Comment:
{MustDelete: false,

Reply:
No need for that if want to set it to false.